composer.json

```diff
@@ -5,27 +5,23 @@
     "license": "proprietary",
     "require": {
         "php": "^8.2",
-        "dvsa/mot-cpms-notifications": "dev-feature-bl-17032-php-82-update",
+        "dvsa/mot-cpms-notifications": "^3.0.0",
         "laminas/laminas-mvc": "^3.3.0",
-        "laminas/laminas-cache": "^3.10.1",
+        "laminas/laminas-cache": "^3.12.0",
         "laminas/laminas-log": "^2.13",
         "laminas/laminas-filter": "^2.13",
         "laminas/laminas-dependency-plugin": "^2.6.0",
-        "laminas/laminas-cache-storage-adapter-filesystem": "^2.3.0",
-        "laminas/laminas-cache-storage-adapter-memory": "^2.2.0",
```

Comment:
Are these versions going backwards?

Reply:
We are just rebasing back to master version